prow/plugins/trigger/pr.go

```diff
@@ -185,20 +190,27 @@ I understand the commands that are listed [here](https://go.k8s.io/bot-commands)
 }
 
 // trustedPullRequest returns whether or not the given PR should be tested.
-// It first checks if the author is in the org, then looks for "/ok-to-test"
-// comments by org members.
-func trustedPullRequest(ghc githubClient, trigger *plugins.Trigger, author, org, repo string, comments []github.IssueComment) (bool, error) {
+// It first checks if the author is in the org, then looks for "ok-to-test" label.
+func trustedPullRequest(ghc githubClient, trigger *plugins.Trigger, author, org, repo string, num int, labels []github.Label) (bool, error) {
 	// First check if the author is a member of the org.
 	if orgMember, err := TrustedUser(ghc, trigger, author, org, repo); err != nil {
 		return false, fmt.Errorf("error checking %s for trust: %v", author, err)
 	} else if orgMember {
 		return true, nil
 	}
+	// Then check if PR has ok-to-test label
```

Comment:
@cjwagner we could fetch labels here (except in ic.go where we directly have ic.Issue.Labels) and save more API tokens for member PRs, what do you think?